### PR TITLE
Release notes 2019-04-25+

### PR DESCRIPTION
**Pull request by Cynerva**
Heyo. I just fixed an issue and wanted to be proactive about making a release note for it. Feels like it isn't done until there's a release note ready to go, or something.

I started a new release section that we can fill in the details of later.

### CODE REVIEW

**Comment by Cynerva** [comment]
I realize @tvansteenburgh usually generates these so maybe this is more disruptive than helpful. I dunno.

**Comment by evilnick** [comment]
True, but if everyone did this @tvansteenburgh wouldn't have to spend so much time fiddling with the output from his script

**Comment by tvansteenburgh** [comment]
FWIW, I'm happy to see this, thanks. It's not going to disrupt what I do re: release notes.

File: pages/k8s/release-notes.md [hunk 1]
```diff
@@ -14,6 +14,50 @@ toc: False
...
+|---------------|-------------------|-------------------------------------------------------|
+| "Never"       | false             | Never use IPIP encapsulation. (The default)           |
+| "Always"      | true              | Always use IPIP encapsulation.                        |
+| "CrossSubnet" | \<Not supported\> | Only use IPIP encapsulation for cross-subnet traffic. |
+
+- Calico BGP support
+
+Several new config options have been added to the Calico charm to support BGP
+functionality within Calico. These additions make it possible to configure
+external peerings, route reflectors, and complex IP pool configurations.
```
Comment (Cynerva) [inline pages/k8s/release-notes.md]: I'd like to add a link here to the [Calico BGP docs](https://github.com/charmed-kubernetes/kubernetes-docs/pull/202), but the CNI pages aren't in yet. @evilnick any advice?

Comment (evilnick) [reply pages/k8s/release-notes.md]: You can add the link based on where they will be, because I intend to merge the CNI stuff before release anyhow

**Pull request merged**